integration_tests/snapshots/logs/http-requests_node14_with_plugin.log

```diff
@@ -1,4 +1,3 @@
-XXXX-XX-XX XX:XX:XX.XXX	XXXX-XXXX-XXXX-XXXX-XXXX	INFO	Snapshot test http requests successfully made to URLs: https://ip-ranges.datadoghq.com,https://ip-ranges.datadoghq.eu
```

Comment:
Comparing with the logs below, I don't think this line should be there. Probably missed from previous integration test updates.